app/components/publishers/vacancies_component.rb

```diff
@@ -43,16 +43,19 @@ def no_jobs_text
   end
 
   def view_applicants(vacancy, card)
-    return unless vacancy.enable_job_applications?
-    return unless include_job_applications?
+    return unless vacancy.enable_job_applications? && include_job_applications?
```

Comment:
The reason I made this in 2 lines is because it is more readable. unless with an and or an or is more complicated to reason in your mind

Reply:
I felt guilty about adding another line so I put this on to one line to compensate for it lol.

I don't think the one line is more readable, so I'll change it back to what it was.

Reply:
It would be more readable if it was an `if`